setup.cfg

```diff
@@ -25,12 +28,14 @@ python_requires = >=3.8
 zip_safe = False
 install_requires =
     click>=7.0.0
-    invenio-records-lom[postgresql,elasticsearch7]>=0.1.0
+    invenio-records-lom>=0.1.0
```

Comment:
probably want `>=0.4.0` here for [these](https://github.com/tu-graz-library/invenio-records-lom/commit/7a4937f313522e5fec0937144ce35dc5d458ddba#diff-9faea468d8735e237545ea4acde4de6e1be0774728b9f0dbf038d2a04508125eR21) changes to `current_records_lom.records_service.create`